Source/Core/DolphinQt2/Config/Graphics/EnhancementsWidget.cpp

```diff
@@ -24,6 +24,8 @@
 #include "VideoCommon/PostProcessing.h"
 #include "VideoCommon/VideoConfig.h"
 
+constexpr const char* DUBOIS_ALGORITHM_SHADER = "dubois";
```

Comment:
This doesn't seem to be used unless the diff is being wonky on me. It's also preferable to use array syntax:

```cpp
constexpr char blah[] = "lol";
```

as it'll only store the string data in the executable, not the string data and a pointer address, like with a const char pointer.